### ISSUE TITLE
Cannot install Joplin 3.2 on Windows

### ISSUE THREAD

**Issue by laurent22** [laurent22/joplin#11508]
> I cannot install 3.2.4 - I get "Joplin cannot be closed. Please close it manually and click Retry to continue." But I did close it manually before installing 3.2.4. I looked through the Task Manager (I'm on Windows 10) and do not see any Joplin processes. I also tried rebooting and running the 3.2.4 install first thing before starting anything else, same result. Any suggestions?

Ref: https://discourse.joplinapp.org/t/desktop-pre-release-v3-2-is-now-available-updated-12-12-2024/41898/23?u=laurent

I think it may be due to this issue? https://github.com/electron-userland/electron-builder/issues/8131

Which has a fix here:

https://github.com/electron-userland/electron-builder/pull/8133

But for now only available on an alpha version. Maybe we can try to use yarn patch to fix this?

**Comment by laurent22** [comment]
Note: it looks like it was working fine in the previous pre-releases?

> Yes, I’ve never had a problem installing the new versions until this one. (I install each new one when the notification comes.) I closed Joplin, checked Task Manager to see if anything was running, rebooted, and got the same message every time. I don’t want to uninstall the existing version that works fine.

Ref: https://discourse.joplinapp.org/t/desktop-pre-release-v3-2-is-now-available-updated-12-12-2024/41898/28?u=laurent

**Comment by personalizedrefrigerator** [comment]
~~This may be related to https://github.com/laurent22/joplin/commit/9e89357e6479a7b6ca9c505869caed50bc4c4c9a.~~

**Comment by laurent22** [comment]
Could it actually be related? This is from one year ago while it seems recent versions < 3.2.4 were working fine

**Comment by personalizedrefrigerator** [comment]
> Could it actually be related? This is from one year ago while it seems recent versions < 3.2.4 were working fine

No, I was wrong about the commit date. Thank you for noticing that!

**Comment by personalizedrefrigerator** [comment]
> Which has a fix here:
>
> https://github.com/electron-userland/electron-builder/pull/8133
>
> But for now only available on an alpha version. Maybe we can try to use yarn patch to fix this?

The [fix](https://github.com/electron-userland/electron-builder/commit/44b04463bf581b4c013586c9010733b518a802a4) seems to be in `electron-builder` >= v25. We're [currently on `electron-builder` v24.13.3](https://github.com/laurent22/joplin/blob/c7e3a31ed80723bd99030afa9e9a7c9fd3cb1739/packages/app-desktop/package.json#L142). [`electron-builder` v25.1.8 seems to be stable](https://www.npmjs.com/package/electron-builder), so upgrading might help.

**Comment by personalizedrefrigerator** [comment]
I'm re-opening this — a user is reporting that 3.2.5 still can't be installed on Windows: https://discourse.joplinapp.org/t/desktop-pre-release-v3-2-is-now-available-updated-18-12-2024/41898/34

**Comment by manderss99** [comment]
confirm, neither 3.2.4 or 3.2.5 installs on windows 11. "Joplin cannot be closed" but have closed Joplin manually before trying to install.

**Comment by manderss99** [comment]
Still having this problem on 3.2.6, windows 11.

**Comment by badabing2005** [comment]
Is there a manual way to workaround this issue?
It's impossible to install updates.
Happens with 3.2.6

**Comment by laurent22** [comment]
Hello, manually uninstalling and installing again seems to work for some users

**Comment by badabing2005** [comment]
> Hello, manually uninstalling and installing again seems to work for some users
Thank you,

Just want to make sure I will not lose any of my settings if I uninstall / reinstall?
Thanks

**Comment by manderss99** [comment]
manually uninstalling 3.2.3 and then installing 3.2.6 worked for me, all settings and plugins was preserved.
all good for now, thanks for an excellent program.

**Comment by badabing2005** [comment]
Thanks, manual uninstall / resinstall worked fine, all settings were preserved.

**Comment by rahji** [comment]
This is still a problem every time I try to install an update. Today I installed 3.6.15 on Windows and the same thing happened - it says that Joplin is running. I quit Joplin from the File menu, I quit it from the taskbar icon. No go. Then I go into the Task manager and try to end the 3 running Joplin tasks. It takes dozens of clicks to get it to finally end the tasks but eventually there is nothing Joplin running. The installer still says that Joplin is running. I did eventually get it to install by clicking Retry (even though nothing was running when it complained).
